### PR TITLE
update bundle methods

### PR DESCRIPTION
**Pull request by itkg-rboessel**

### CODE REVIEW

**Comment by itkg-rboessel** [comment]
@nicolasThal @itkg-ngilain @itkg-nanne @itkg-alavieille

File: ApiBundle/Controller/ApiClientController.php [hunk 1]
```diff
@@ -29,7 +28,7 @@ class ApiClientController extends BaseController
      *
      * @Api\Serialize()
      *
-     * @return FacadeInterface
+     * @return \OpenOrchestra\BaseApi\Facade\FacadeInterface
```
Comment (nicolasThal) [inline ApiBundle/Controller/ApiClientController.php]: I would rather keep the `use` and only the class name in the @return

File: ApiBundle/Controller/AreaController.php [hunk 2]
```diff
@@ -32,7 +31,7 @@ class AreaController extends BaseController
      *
      * @Api\Serialize()
      *
-     * @return FacadeInterface
+     * @return \OpenOrchestra\BaseApi\Facade\FacadeInterface
```
Comment (nicolasThal) [inline ApiBundle/Controller/AreaController.php]: same

File: ApiBundle/Controller/AreaController.php [hunk 3]
```diff
@@ -54,7 +53,7 @@ public function showInNodeAction($areaId, $nodeId)
...
```
Comment (nicolasThal) [inline ApiBundle/Controller/AreaController.php]: sam

File: ApiBundle/Controller/BlockController.php [hunk 4]
```diff
@@ -25,7 +24,7 @@ class BlockController extends BaseController
      *
      * @Config\Security("has_role('ROLE_ACCESS_TREE_NODE')")
      *
-     * @return FacadeInterface
+     * @return \OpenOrchestra\BaseApi\Facade\FacadeInterface
```
Comment (nicolasThal) [inline ApiBundle/Controller/BlockController.php]: same

File: ApiBundle/Controller/ContentController.php [hunk 5]
```diff
@@ -38,7 +36,8 @@ class ContentController extends BaseController
      *
      * @Api\Serialize()
      *
-     * @return FacadeInterface
+     * @return \OpenOrchestra\BaseApi\Facade\FacadeInterface
```
Comment (nicolasThal) [inline ApiBundle/Controller/ContentController.php]: same

File: ApiBundle/Controller/ContentController.php [hunk 6]
```diff
@@ -63,7 +62,8 @@ public function showAction(Request $request, $contentId)
...
```
Comment (nicolasThal) [inline ApiBundle/Controller/ContentController.php]: same

File: ApiBundle/Controller/ContentController.php [hunk 7]
```diff
@@ -112,7 +113,7 @@ protected function showOrCreate(Request $request, $contentId)
      *
      * @Api\Groups({GroupContext::G_HIDE_ROLES})
      *
-     * @return FacadeInterface
+     * @return \OpenOrchestra\BaseApi\Facade\FacadeInterface
```
Comment (nicolasThal) [inline ApiBundle/Controller/ContentController.php]: same

File: ApiBundle/Controller/ContentTypeController.php [hunk 8]
```diff
@@ -31,7 +30,7 @@ class ContentTypeController extends BaseController
      *
      * @Api\Serialize()
      *
-     * @return FacadeInterface
+     * @return \OpenOrchestra\BaseApi\Facade\FacadeInterface
```
Comment (nicolasThal) [inline ApiBundle/Controller/ContentTypeController.php]: same

File: ApiBundle/Controller/ContentTypeController.php [hunk 9]
```diff
@@ -50,7 +49,7 @@ public function showAction($contentTypeId)
...
```
Comment (nicolasThal) [inline ApiBundle/Controller/ContentTypeController.php]: same

**Pull request closed without merging**